modules/social_features/social_group/modules/social_group_request/src/Form/GroupRequestMembershipRequestForm.php

```diff
@@ -2,90 +2,46 @@
 
 namespace Drupal\social_group_request\Form;
 
-use Drupal\Core\Cache\CacheTagsInvalidatorInterface;
-use Drupal\Core\Entity\EntityTypeManagerInterface;
-use Drupal\Core\Form\FormBase;
+use Drupal\Core\Entity\ContentEntityForm;
 use Drupal\Core\Form\FormStateInterface;
-use Drupal\Core\Session\AccountInterface;
-use Drupal\Core\StringTranslation\TranslationInterface;
 use Drupal\grequest\Plugin\Group\Relation\GroupMembershipRequest;
 use Drupal\group\Entity\GroupInterface;
 use Symfony\Component\DependencyInjection\ContainerInterface;
 
 /**
  * Provides a form to request group membership.
  */
-class GroupRequestMembershipRequestForm extends FormBase {
+class GroupRequestMembershipRequestForm extends ContentEntityForm {
 
   /**
-   * Group entity.
+   * The module handler.
    *
-   * @var \Drupal\group\Entity\GroupInterface
+   * @var \Drupal\Core\Extension\ModuleHandlerInterface
    */
-  protected $group;
+  protected $moduleHandler;
 
   /**
-   * Group membership request.
-   *
-   * @var \Drupal\group\Entity\GroupRelationshipInterface
-   */
-  protected $groupContent;
```

Comment:
This change is irrelevant to this PR as this was not previously used, let's split this change on a separated PR